setup.py

```diff
@@ -127,10 +127,12 @@ def run(self):
         'Intended Audience :: Developers',
         'Natural Language :: English',
         'Operating System :: OS Independent',
-        'Programming Language :: Python :: 2.7',
-        'Programming Language :: Python :: 3.5',
-        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3 :: Only',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
```

Comment:
You've commented 3.10 in travis, so probably should comment it here too so everything matches

Reply:
Yes.. Maybe they will integrate 3.10 in the mean time...